spec/lib/rbac/filterer_spec.rb

```diff
@@ -1244,17 +1244,29 @@ def get_rbac_results_for_and_expect_objects(klass, expected_objects)
       end
 
       context "with VMs and Templates" do
+        let(:root) do
+          datacenters = FactoryBot.create(:ems_folder, :name => "Datacenters")
+          datacenters.parent = @ems
+          datacenters
+        end
+
+        let(:dc) do
+          datacenter = FactoryBot.create(:ems_folder, :name => "Datacenter1")
```

Comment:
Shouldn't his be ` datacenter = FactoryBot.create(:datacenter, :name => "Datacenter1")`?

Reply:
thanks @gtanzillo 

we are using for datacenter  it EmsFolder - it is inherited from "belongsto filter sentence":

```
/belongsto/ExtManagementSystem|vCenter/EmsFolder|Datacenters/EmsFolder|EMEA CloudLab/EmsFolder|host/EmsCluster|Infrastructure
```
and there is used `EmsFolder` for datacenter.